### DISCUSSION TITLE
Metal Slug 6

### DISCUSSION THREAD

**Discussion by 3lc3z4r** [flyinghead/flycast#652] [Ideas]
I wonder if it is possible to take into account a small update to fix this minor error in metal slug 6 (exactly the loading screen is bugged, normally it should say "now loading")

![imagen_2022-05-30_181612248](https://user-images.githubusercontent.com/106557341/171067168-93afb59f-4fd7-4df2-bcf3-c1ee92632049.png)

I mean something like that would be the right thing ,on the other hand the same emulation if it is the best, thanks
![imagen_2022-05-30_181949134](https://user-images.githubusercontent.com/106557341/171067361-2ea7f57f-4995-4656-9263-4505e20de253.png)

**Comment by flyinghead** [comment]
This is due to Flycast detecting a direct write to the framebuffer, possibly caused by a VRam initialization by the BIOS or the game. I haven't been able to reproduce it with mslug6 but it happens with a number of Atomiswave games.

Minor issue and no easy fix.

**Reply by 3lc3z4r** [reply]
well for now as it seems it is difficult to fix it, it only remains to wait.
In another case, I have seen that the emulator in a very old version did not present this error (I found that proof in a youtube video around 2021 I think, where it was running normally)

this is the video: 
https://www.youtube.com/watch?v=BzOhkET4RnU
